emm-docs/content/subcommands/pull-request.md

```diff
@@ -31,3 +31,7 @@ you must also provide a `--title`.
 ```bash
 $ evg-module-manager pull-request --title "my pull request title" --body "my pull request body"
 ```
+
+Note: Before using this command, you may want to confirm your git remote url. The
+default remote this command using is `origin`. If you want to create the pull request
```

Comment:
I think we should be more direct with the phrasing here. We should also provide some guidance on how check/update remotes.

Maybe something like:

Note: This command will create a pull request against the `origin` repository in github. You can run `git remote show origin` to confirm this is the repository you would like to use.
```
$ git remote show origin
* remote origin
  Fetch URL: git@github.com:mongodb/mongo.git
  Push  URL: git@github.com:mongodb/mongo.git
  HEAD branch: master
  Remote branches:
...
```

You can change the `origin` repository with the `git remote set-url origin <new_remote_url>` command.